src/AdvApi32.Desktop/AdvApi32.Desktop.csproj

```diff
@@ -65,6 +65,11 @@
     <Compile Include="AdvApi32+ServiceControlAction.cs" />
     <Compile Include="AdvApi32+ServiceFailureActions.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="AdvApi32.exports.txt">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
The value of `Always` defeats the Visual Studio up-to-date check for builds. If you build the same project twice, it will actually run the build twice instead of reporting that the project is up-to-date the second time. Can you change all of these to `PreserveNewest`?